executor/ddl.go

```diff
@@ -431,6 +442,15 @@ func (e *DDLExec) executeDropSequence(s *ast.DropSequenceStmt) error {
 	return e.dropTableObject(s.Sequences, sequenceObject, s.IfExists)
 }
 
+func (e *DDLExec) getLocalTemporaryTable(schema model.CIStr, table model.CIStr) (table.Table, bool) {
+	temporaryTables := e.ctx.GetSessionVars().LocalTemporaryTables
```

Comment:
Maybe it's a better idea to unexport `LocalTemporaryTable`  ...
Some of the directly operation on this field makes the code harder to maintain in the future.

It should have been possible to use

```
schema := e.ctx.GetInfoSchema()
schema.TableByName()
```

Reply:
LocalTemporaryTable still has to be exported because create/drop table are using it to add or remove table from meta. But I'm fine to get temporary a table by `e.ctx.GetInfoSchema()`.